src/views/projects/Issue/New.svelte

```diff
@@ -0,0 +1,283 @@
+      tagCaption = "This tag already exists";
+      return;
+    }
+    tags.push(tag);
+    tags = tags;
+    tag = "";
+  }
+
+  let issueTitle = "";
+  let issueText = "";
+  let assignees: string[] = [];
+  let assignee: string = "";
+  let assigneeCaption: string | undefined = undefined;
+  let tags: string[] = [];
+  let tag: string = "";
+  let tagCaption: string | undefined = undefined;
+
+  function navigateToIssues() {
+    router.push({
+      resource: "projects",
+      params: {
+        id: project.id,
+        seed: project.seed.addr.host,
+        view: {
+          resource: "issues",
+        },
+      },
+    });
+  }
+
+  async function createIssue() {
+    try {
+      await Issue.createIssue(
+        project.id,
+        issueTitle,
+        issueText,
+        assignees,
+        tags,
+        project.seed.addr,
+        session.id,
+      );
+      navigateToIssues();
+    } catch {
+      modal.show({
+        component: AuthenticationErrorModal,
+        props: {
+          title: "Authentication failed",
+          subtitle: [
+            "Could not create the issue. Make sure you're still logged in.",
+          ],
+        },
+      });
+    }
+  }
+</script>
+
+<style>
+  main {
+    padding: 0 2rem 0 8rem;
+  }
+  .form {
+    display: grid;
+    grid-template-columns: minmax(0, 2fr) 1fr;
+    margin-bottom: 1rem;
+  }
+  .side-bar {
+    border-left: 1px solid var(--color-foreground-4);
+    padding-left: 1rem;
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+  }
+  .actions {
+    text-align: right;
+    display: flex;
+    justify-content: flex-end;
+    flex-direction: row;
+    gap: 1rem;
+    margin-top: 1rem;
+  }
+  .editor {
+    flex: 2;
+    padding-right: 1rem;
+  }
+  .section {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    margin-bottom: 3rem;
+  }
+  .section:not(:first-child) {
+    border-top: 1px solid var(--color-foreground-4);
+    padding-top: 1rem;
+  }
+  .summary {
+    padding: 1rem;
+    background: var(--color-foreground-1);
+    border-radius: var(--border-radius);
+    margin-bottom: 2rem;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+  }
+  .chips {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    gap: 0.5rem;
+  }
+  .tags {
+    flex-direction: row;
+  }
+  .tag {
+    max-width: 15rem;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+  .assignee {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+  @media (max-width: 960px) {
+    main {
+      padding-left: 2rem;
+    }
+  }
+  @media (max-width: 720px) {
+    .form {
+      grid-template-columns: minmax(0, 1fr);
+    }
+  }
+</style>
+
+<main>
+  <div class="form">
+    <div class="editor">
+      {#if preview}
+        <div class="summary txt-medium" class:txt-missing={preview}>
+          {issueTitle || "No title"}
```

Comment:
```suggestion
          {issueTitle ?? "No title"}
```

Reply:
In the end I changed this to be more similar to how we handle it in `Comment` to trim the `issueTitle` and check strict equal against an empty string